src/components/mobile-menu-levels/level-components/index.tsx

```diff
@@ -0,0 +1,79 @@
+// Components
+import {
+	SidebarHorizontalRule,
+	SidebarNavMenuItem,
+} from 'components/sidebar/components'
+// Utils
+import { generateTopLevelSubNavItems } from 'lib/generate-top-level-sub-nav-items'
+import { generateProductLandingSidebarMenuItems } from 'components/sidebar/helpers/generate-product-landing-nav-items'
+import { generateResourcesNavItems } from 'components/sidebar/helpers'
+// Types
+import type { MenuItem } from 'components/sidebar'
+import type { ProductData, ProductSlug } from 'types/products'
+import type { MobileMenuLevelData } from '../types'
+// Styles
+import s from './level-components.module.css'
+
+/**
+ * Render a list of resource nav items for a given product slug.
+ */
+export function ProductResourceNavItems({ slug }: { slug: ProductSlug }) {
+	return (
+		<ul className={s.listResetStyles}>
+			{generateResourcesNavItems(slug).map((item: MenuItem, index: number) => (
+				// eslint-disable-next-line react/no-array-index-key
+				<SidebarNavMenuItem item={item} key={index} />
+			))}
+		</ul>
+	)
+}
+
+/**
+ * Return mobile menu level data for the shared `main` menu pane.
+ */
+export function mobileMenuLevelMain(): MobileMenuLevelData {
```

Comment:
The `mobileMenuLevelMain` and `mobileMenuLevelProduct` are intended to be re-usable chunks of content we consistently in mobile menus. The analogous abstraction that currently exists is how we use `generateTopLevelSubNavItems` and `generateProductLandingSidebarMenuItems`, but with our `Sidebar` component (which actually renders both literal "sidebar" contents, and mobile menu panes).